<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一键设置环境变量</string>
  </property>
  <widget class="QTabWidget" name="tab_container">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="environment_tab">
    <attribute name="title">
     <string>环境变量</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>环境变量</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="key_value_path">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>1011</width>
       <height>421</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>add_path</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>JAVA_HOME</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>D:\DevelopmentTools\Softare\GreenSoftWare\a-language\jdk\Java1.8</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>\bin;</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="add_row">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>460</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>增加1行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_select_row">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>460</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>删除选中的行</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="maven_tab">
    <attribute name="title">
     <string>maven配置</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>本地Maven仓库地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="local_repository">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>代理设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>active</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>protocol</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>host</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxy_id">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxy_active">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxy_protocol">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxy_host">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxy_port">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>镜像设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>url</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>mirrorOf</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mirror_id">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mirror_name">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mirror_url">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mirror_mirrorOf">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="proxy_checkbox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>80</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>nodejs配置</string>
    </attribute>
    <widget class="QPlainTextEdit" name="npmrc_config">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>501</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nodejs家目录/etc/npmrc</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_environment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>创建表格内全部环境变量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_environment">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除表格内全部环境变量</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
